### ISSUE TITLE
Variants of QUAL 0 have FILTER Value of PASS

### ISSUE THREAD

**Issue by DarioS** [KCCG/mity#13]
I am exploring Mity output. What do these mean?

```
chrM    383     .       T       A,G     0       PASS
chrM    385     .       A       C       0       PASS
chrM    387     .       C       A       1.80    SBA
chrM    391     .       T       A       0       PASS
chrM    394     .       C       A       10.10   PASS

```

There should perhaps be a couple of sentences explaining it added to the user guide.

**Comment by drmjc** [comment]
Hi Dario,
Thanks for reporting this issue. It's not one I've noticed, so can you provide a reproducible example?

**Comment by DarioS** [comment]
Commands are in the header section. See lines beginning with `##freebayesCommandline` and `##mityCommandline` of [test.txt](https://github.com/KCCG/mity/files/8241982/test.txt). File extenstion renamed from vcf to txt to allow upload to GitHub, but it's really a VCF file. Only QUAL 0 and PASS shown.
.

**Comment by drmjc** [comment]
I see a couple of things that could cause this. First I think you've
supplied a cram, which mity doesn't officially support and second I see
three samples in the VCF file - do you have three different samples within
the same CRAM?

**Comment by DarioS** [comment]
Yes, input was CRAM format. The Garvan Institute core facility found BAM and CRAM format produced the same output in testing and applies mity to CRAM files as part of their production pipeline. No, each sample is in one VCF. They use GATK's CombineVariants, according to the header:

```
##GATKCommandLine.CombineVariants ... [(RodBinding name=variant source=11_21_PK6_Clone2.mity.vcf.gz), (RodBinding name=variant2 source=11_21_PK6_Clone4.mity.vcf.gz), (RodBinding name=variant3 source=11_21_PK6_Fibro.mity.vcf.gz)])] out=/paella/CRAMPipeline/projects/RYADAV/R_211102_RYADAV_DNA_M001/mity/work/4c/2e9edf3447194cdb12b0c3321ceb3a/R_211102_RYADAV_DNA_M001.mity.vcf.gz ...
```

**Comment by trentzz** [comment]
Hi, the issues you've mentioned should be resolved in the new release of Mity 1.1.0. If you encounter any further problems, please feel free to open a new issue. Thanks!
